packages/vertexai/src/requests/response-helpers.ts

```diff
@@ -65,19 +68,17 @@ export function addHelpers(
         );
       }
       if (hadBadFinishReason(response.candidates[0])) {
-        throw ERROR_FACTORY.create(VertexError.RESPONSE_ERROR, {
-          message: `${formatBlockErrorMessage(response)}`,
-          response
-        });
+        throw new VertexAIError(
+          VertexAIErrorCode.RESPONSE_ERROR,
+          `${formatBlockErrorMessage(response)}`
```

Comment:
This should be formatted with "Response error: ${formatBlockErrorMessage(response}..." etc like the one on line 45. If you find yourself having to type the same formatting template multiple times like this, it might be worth making a helper function in src/error.ts named `createResponseBlockError()` or something that takes blockErrorMessage as an arg or something and returns the VertexAI error with the formatted string applied. It looks like it's only used twice so not sure if it's worth it at this point.

Reply:
Done